easybuild/easyconfigs/s/sratoolkit/sratoolkit-2.5.2.eb

```diff
@@ -0,0 +1,46 @@
+
+easyblock = 'PackedBinary'
+
+name = 'sratoolkit'
+version = '2.5.2'
+
+homepage = 'http://www.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?view=software'
+description = """The SRA Toolkit, and the source-code SRA System Development Kit (SDK), will allow you to 
+programmatically access data housed within SRA and convert it from the SRA format"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://ftp-trace.ncbi.nlm.nih.gov/sra/sdk/%(version)s/']
+sources = ['sratoolkit.%(version)s-ubuntu64.tar.gz']
+
+executables = [
+'abi-dump', 'abi-dump.2', 'abi-dump.%(version)s', 'abi-load', 'abi-load.2', 'abi-load.%(version)s', 'align-info', 
+'align-info.2', 'align-info.%(version)s', 'bam-load', 'bam-load.2', 'bam-load.%(version)s', 'blastn_vdb', 
+'blastn_vdb.2', 'blastn_vdb.2.2.31-%(version)s', 'cache-mgr', 'cache-mgr.2', 'cache-mgr.%(version)s', 'cg-load', 
+'cg-load.2', 'cg-load.%(version)s', 'fastq-dump', 'fastq-dump.2', 'fastq-dump.%(version)s', 'fastq-load', 
+'fastq-load.2', 'fastq-load.%(version)s', 'helicos-load', 'helicos-load.2', 'helicos-load.%(version)s', 
+'illumina-dump', 'illumina-dump.2', 'illumina-dump.%(version)s', 'illumina-load', 'illumina-load.2', 
+'illumina-load.%(version)s', 'kar', 'kar.2', 'kar.%(version)s', 'kdbmeta', 'kdbmeta.2', 'kdbmeta.%(version)s', 
+'latf-load', 'latf-load.2', 'latf-load.%(version)s', 'ncbi', 'pacbio-load', 'pacbio-load.2', 'pacbio-load.%(version)s',
+'prefetch', 'prefetch.2', 'prefetch.%(version)s', 'rcexplain', 'rcexplain.2', 'rcexplain.%(version)s', 'remote-fuser',
+'remote-fuser.2', 'remote-fuser.%(version)s', 'sam-dump', 'sam-dump.2', 'sam-dump.%(version)s', 'sff-dump', 
+'sff-dump.2', 'sff-dump.%(version)s', 'sff-load', 'sff-load.2', 'sff-load.%(version)s', 'sra-kar', 'sra-kar.2', 
+'sra-kar.%(version)s', 'sra-pileup', 'sra-pileup.2', 'sra-pileup.%(version)s', 'sra-sort', 'sra-sort.2', 
+'sra-sort.%(version)s', 'sra-stat', 'sra-stat.2', 'sra-stat.%(version)s', 'srapath', 'srapath.2', 
+'srapath.%(version)s', 'srf-load', 'srf-load.2', 'srf-load.%(version)s', 'tblastn_vdb', 'tblastn_vdb.2', 
+'tblastn_vdb.2.2.31-%(version)s', 'test-sra', 'test-sra.2', 'test-sra.%(version)s', 'vdb-config', 'vdb-config.2', 
+'vdb-config.%(version)s', 'vdb-copy', 'vdb-copy.2', 'vdb-copy.%(version)s', 'vdb-decrypt', 'vdb-decrypt.2', 
+'vdb-decrypt.%(version)s', 'vdb-dump', 'vdb-dump.2', 'vdb-dump.%(version)s', 'vdb-encrypt', 'vdb-encrypt.2', 
+'vdb-encrypt.%(version)s', 'vdb-lock', 'vdb-lock.2', 'vdb-lock.%(version)s', 'vdb-passwd', 'vdb-passwd.2', 
+'vdb-passwd.%(version)s', 'vdb-unlock', 'vdb-unlock.2', 'vdb-unlock.%(version)s', 'vdb-validate', 'vdb-validate.2', 
+'vdb-validate.%(version)s'
+]
+
+files_to_copy = [(executables, 'bin/')]
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in executables],
```

Comment:
if we use glob patterns (with `*`) in `executables`, we obviously won't be able to use `executables` here